scripts/analyze.py

```diff
@@ -0,0 +1,131 @@
+########### Python Form Recognizer Async Analyze #############
+import json
+import time
+import getopt
+import sys
+import os
+from requests import get, post
+
+def main(argv):
+    input_file, file_type = getArguments(argv)
+    runAnalysis(input_file, file_type)
```

Comment:
Should we use "prediction" instead of "analysis" to keep the terminology the same with the UI? In the Quickstart documentation, we use the term "analyze," but in the UI we use "predict"

Reply:
let's use analysis